server/guiserver/tests/bundles/test_views.py

```diff
@@ -609,21 +609,26 @@ def test_valid_yaml(self):
         expected_response = {
             'Response': {
                 'Changes': [
-                    {'args': ['cs:trusty/django-42'],
-                     'requires': [],
-                     'id': 'addCharm-0',
-                     'method': 'addCharm'},
-                    {'args': ['cs:trusty/django-42', 'django', {}],
-                     'requires': ['addCharm-0'],
-                     'id': 'addService-1',
-                     'method': 'deploy'},
-                    {'args': ['$addService-1', 1, None],
-                     'requires': ['addService-1'],
-                     'id': 'addUnit-2',
-                     'method': 'addUnit'},
-                ],
-            },
+                    {
+                        u'args': ['cs:trusty/django-42'],
+                        u'requires': [],
+                        u'id': u'addCharm-0',
+                        u'method': u'addCharm'
+                    }, {
+                        u'args': [u'$addCharm-0', 'django', {}, u'', {}],
```

Comment:
Could you please make sure the GUI already handles these two new deploy arguments?